doc/MAGIC.md

```diff
@@ -60,6 +60,8 @@ For example, this is the formula for spell failure chance:
 
 ```( ( ( ( spell_level - spell_difficulty ) * 2 + intelligence + spellcraft_skill ) - 30 ) / 30 ) ^ 2```
 
+(Just to make it clear, this is not the resultant value of failure chance, for more detail, see function spell_fail in magic.cpp.)
```

Comment:
this is misleading. instead it should mention that it caps out at an effective spell level of 30 at 0% failure chance.

Reply:
I think it not really necessary to write the complete detial of the calculation because it will be a long paragraph (to get the final value of failure chance, the process doesn't only include the if-then-else of 30-cap and 0-floor) . And I don't think it is misleading, it is true that the formula's result is actually not failure chance, just a medium value. Comparing to the original document which only state the formula, the new edition tells that there are more things to be processed to get the final failure chance value, and also provide a guide to find where the function is, obviously much better.

Reply:
Nevermind, maybe I can just write " the actual calculation doesn't really look like this" in stead of " this is not the resultant value of failure chance".